### ISSUE TITLE
A11y: "Find Properties" should be marked up as a link

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/vacant-lots-proj#298]
## Description

"Find Properties" is currently a button. It should be marked up as a link to preserve its semantic meaning since it takes you to a different URL.

<img width="1217" alt="Screenshot 2024-01-30 at 16 11 19" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/fec2f585-f906-4c49-9372-a23a5c6b4c46">
